app/markets/ActivityTable.tsx

```diff
@@ -18,16 +23,12 @@ import { useQuery } from '@tanstack/react-query';
 import { cx } from 'class-variance-authority';
 import { useState } from 'react';
 
-const getTagColorScheme = (transactionType: TransactionType): TagColorSchemeProp => {
-  switch (transactionType) {
-    case TransactionType.Buy:
+const getTagColorScheme = (outcome: string): TagColorSchemeProp => {
+  switch (outcome) {
+    case 'Yes':
       return 'success';
-    case TransactionType.Sell:
+    case 'No':
       return 'danger';
```

Comment:
Let's use outcome index here. There are markets where outcome string isn't "yes" or "no"

Reply:
You're right! I'll take the omen types here. Thanks!